fe1-web/src/core/components/DropdownSelector.tsx

```diff
@@ -13,26 +15,38 @@ import containerStyles from '../styles/stylesheets/containerStyles';
  *  - Default selected option (not required, if not specified then first in the array is chosen)
  */
 
-const DropdownSelector = (props: IPropTypes) => {
-  const { selected } = props;
-  const { onChange } = props;
-  const { values } = props;
-  const options: any = [];
-  values.forEach((value) => options.push(<Picker.Item key={value} label={value || ''} />));
+const DropdownSelector = (
+  props: ExtendType<
+    IPropTypes,
+    {
+      // make the type for 'onChange' more concrete than the inferred type
+      onChange: (itemValue: string | null, itemIndex: number) => void;
```

Comment:
I would love that too, but unfortunately, prop-types are a runtime construct in React.
They have no notion of TypeScript types because that only exists at compile-time, in JavaScript.
JavaScript is not strongly typed, so it'd be impossible to extract the right types from the function, even if it were possible to indicate to prop-types its expected parameters and return value.